docs/samples/v1beta1/catboost/README.md

```
@@ -0,0 +1,56 @@
+
+## Create the InferenceService
+
+Apply the CRD:
+
+```bash
+kubectl apply -f catboost.yaml
+```
+
+Expected Output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)